h2/settings.py

```diff
@@ -254,12 +254,18 @@ def __len__(self):
         return len(self._settings)
 
 
```

Comment:
This whitespace should probably get removed. Only supposed to be 1 line between class methods.